addons/web_editor/static/src/js/backend/convert_inline.js

```diff
@@ -810,9 +810,12 @@ function fontToImg($editable) {
             font.before(wrapper);
             font.remove();
             wrapper.style.setProperty('padding', padding);
-            wrapper.style.setProperty('width', width + 'px');
+            const wrapperWidth = width + ['left', 'right'].reduce((sum, side) => (
+                sum + (+_getStylePropertyValue(image, `margin-${side}`).replace('px', '') || 0)
```

Comment:
Any chance you could get a `NaN` here ? Just making sure.

Reply:
Yes, that's why I put `|| 0`.